src/Symfony/Component/Clock/Clock.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Clock;
+
+use Psr\Clock\ClockInterface;
+
+/**
+ * A global clock.
```

Comment:
```suggestion
 * A global clock singleton.
```

Reply:
I'd prefer not adding the word singleton.